server/src/shared/app/controller-without-dto.ts

```diff
@@ -0,0 +1,10 @@
+import BaseController from './base-controller'
+import { UseCaseWithoutDTO } from './use-case-without-dto'
+
+export abstract class ControllerWithoutDTO<
```

Comment:
It still bothers me that we're naming a class after what it isn't, rather than what it is. It's a sign that something went wrong during design, but our path to this implementation seemed pretty much inevitable?